types/auth0/index.d.ts

```diff
@@ -1277,6 +1289,17 @@ export class ManagementClient<A=AppMetadata, U=UserMetadata> {
 
   deleteCustomDomain(params: ObjectWithId): Promise<void>;
   deleteCustomDomain(params: ObjectWithId, cb: (err: Error) => void): void;
+
+  // User enrollment
+  getGuardianEnrollments(): Promise<Enrollment[]>;
+  getGuardianEnrollments(cb: (err: Error, response: Enrollment[]) => void): void;
+
+
+  getGuardianEnrollment(params: ObjectWithId): Promise<Enrollment[]>;
+  getGuardianEnrollment(params: ObjectWithId, cb: (err: Error, response: Enrollment[]) => void): void;
```

Comment:
are those supposed to return a list? or single object?
https://github.com/auth0/node-auth0/blob/c509610693ccec51fb17c586eafe57c6e708e7d2/src/management/GuardianManager.js#L158

Reply:
It should return an Enrollment list. I already update it and now it is working clear with my project.
Also I realized that I included a unnecessary method getGuardianEnrollment and getGuardianEnrollments (in plural), I update this and the singular method is no longer needed

By the way Thanks for the review, 
I will be aware of your reply